tutorials/scripting/c_sharp/c_sharp_signals.rst

```diff
@@ -30,8 +36,13 @@ In addition, you can always access signal names associated with a node type thro
 .. warning::
 
     While all engine signals connected as events are automatically disconnected when nodes are freed, custom
-    signals aren't. Meaning that: you will need to manually disconnect (using ``-=``) all the custom signals you
-    connected as C# events (using ``+=``).
+    signals connected using ``+=`` aren't. Meaning that: you will need to manually disconnect (using ``-=``)
```

Comment:
The "as events" on the line above already implies the "connected using `+=`" for me, that's why it was worded like this.

Reply:
I think this is ambiguous:

> While all engine signals connected as events are automatically disconnected when nodes are freed, custom signals aren't.

While all (engine signals connected as events) are automatically disconnected [...], (custom signals) aren't.
vs.
While all (engine signals) (connected as events) are automatically disconnected [...], (custom signals) (connected as events [implied]) aren't.

I don't know if that's enough to convey what I mean, but I didn't even consider the second interpretation--I thought this line was entirely mistaken when I first read it. 😄

Reply:
Yeah, the whole sentence is too complicated for me either way, so I don't really have a strong opinion. I hate we don't have a simple way to convey `+=`.

Reply:
I'll take another look later with fresh eyes... yeah, pulling apart the complexity a bit is probably the right move.

And yeah, this is all already a nightmare in casual convos, getting it right in a doc seems like it may be tough. 😅

Reply:
After playing with the behavior a bit, I found a whole new chapter to this about capturing variables, so ended up writing a decent amount at https://github.com/godotengine/godot-docs/pull/8815.